CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2293,15 +2344,18 @@ protected override void OnMouseWheel(MouseWheelEventArgs e)
                 var point = e.GetPosition(this);
                 var modifiers = e.GetModifiers();
                 var isShiftKeyDown = Keyboard.IsKeyDown(Key.LeftShift) || Keyboard.IsKeyDown(Key.RightShift);
-                var pointX = (int)point.X;
-                var pointY = (int)point.Y;
 
-                browser.SendMouseWheelEvent(
-                    pointX,
-                    pointY,
-                    deltaX: isShiftKeyDown ? e.Delta : 0,
-                    deltaY: !isShiftKeyDown ? e.Delta : 0,
-                    modifiers: modifiers);
+                if (!mouseTeleport.IsInsideOriginalRect((int)point.X, (int)point.Y))
```

Comment:
Under what instance do we want to ignore the mouse wheel?

These methods should simple call `GetAdjustedMouseCoords`, no logic should be in any of these methods.

Reply:
I moved the logic to the class.